lib/Bindings/Python/RTLOps.td

```diff
@@ -0,0 +1,15 @@
+//===-- RTLOps.td - Entry point for RTLOps bindings --------*- tablegen -*-===//
```

Comment:
Is this file going to grow for the RTL dialect? If not, we will probably want to stuff all the dialects in here. No need to change now, just food for thought.

Reply:
It may grow a bit, but you are right, this is probably about it. However, unless something changes upstream, we will probably want to have several small files like this. This is the PYTHON_BINDING_TD_FILE mentioned here: https://mlir.llvm.org/docs/Bindings/Python/#integration-with-ods. I would expect one small one per dialect, similar to the .td files upstream: https://github.com/llvm/llvm-project/tree/main/mlir/lib/Bindings/Python